generator/test/resources/examples/apidoc.json

```diff
@@ -656,7 +656,7 @@
           "type": "visibility",
           "description": "Public organizations will be listed in apidoc directory. Organizations with visibility organization will only be visible to members of that org.",
           "default": "organization",
-          "required": false,
+          "required": true,
```

Comment:
This change is important. I believe it implies that we would need to change the api.json for apidoc to make this PR work (though not the spec).